src/main/java/de/retest/surili/commons/actions/NavigateToAction.java

```diff
@@ -19,4 +19,9 @@
 		return Optional.empty();
 	}
 
+	@Override
+	public String toString() {
+		return NavigateToAction.class.getSimpleName() + "(" + url + ")";
```

Comment:
To pick up the keys / field names argument from above: Here you override the Lombok `toString()` method just to omit `url=`. Worth it?

Also above, e.g., `NavigateRefreshAction`: Lombok generates `NavigateRefreshAction()`, your override `NavigateRefreshAction`. Worth it?

Reply:
Thanks, I wasn't aware that `@Value` incorporates `@ToString`.